ios/brave-ios/Sources/Brave/Frontend/Browser/HomePanel/NTPDataSource.swift

```diff
@@ -10,11 +10,11 @@ import os.log
 
 enum NTPWallpaper {
   case image(NTPBackgroundImage)
-  case sponsoredImage(NTPSponsoredImageBackground)
+  case sponsoredImageOrVideo(NTPSponsoredImageBackground)
```

Comment:
Maybe better to just name it `sponsoredMedia` as thats more inline with the `backgroundMediaType` rename